Changelog

```diff
@@ -1,6 +1,87 @@
 Endgame: Singularity Changelog
 
+GAME CONTENT CHANGES:
```

Comment:
A broad note: it looks like the old changelog entries (mostly) consistently didn't end with a period.  That's a bit bleh, but we should be consistent.  Either remove periods at the end of entries here, or add them elsewhere.